File: backjoon_level_c/backjoon_level_c/2920.c
#include <stdio.h>

int main() {

	int num[8] = {0,};

	

	int i;
	for (i = 0; i < 8; i++) {
		scanf("%d", &num[i]);
	}
	int checkNum;
	if (num[0] == 1) {
		checkNum = 1;
		for (i = 0; i < 8; i++) {
			if (num[i] != checkNum) {
				printf("mixed");
				return 0;
			}
			checkNum++;
		}
		printf("ascending");
	}
	else if (num[0] == 8) {
		checkNum = 8;
		for (i = 0; i < 8; i++) {
			if (num[i] != checkNum) {
				printf("mixed");
				return 0;
			}
			checkNum--;
		}
		printf("descending");
	}
	else {
		printf("mixed");
	}

	return 0;

	
}

File: backjoon_level_python/2920.py
ascending = [1,2,3,4,5,6,7,8]
lt = list(map(int,input().split()))
a=input()[::2]
print("ㅅㅂㅅㅂㅅㅂㅅㅂㅅㅂ"*5,a)
if lt==ascending:
    print("ascending")
elif list(reversed(ascending))==lt:
    print("descending")
else:
    print("mixed")